src/utils/parseError.ts

```diff
@@ -158,12 +162,38 @@ export const parseError = async (
 
   switch (e.code) {
     case 'CALL_EXCEPTION':
-      return new ProviderError(
-        LifiErrorCode.TransactionFailed,
-        e.reason,
-        await getTransactionNotSentMessage(step, process),
-        e.stack
-      )
+      try {
+        if (!step?.action.fromChainId) {
+          throw new Error('Signer is not defined.')
+        }
+
+        const response = await fetchTxErrorDetails(
+          e.transactionHash,
+          step?.action.fromChainId
+        )
```

Comment:
This method is async only because of ChainService, it's not really so async 😄 We should not use tenderly here and we can determine errors without it.